builders/ethereum/libraries/ethersrs.md

```diff
@@ -606,8 +606,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 If successful, the transaction receipt will be displayed in the terminal. You can use the `read_number` function in the `main` function to make sure that value is changing as expected. If you're using the `read_number` function after resetting the number, you should see `0` printed to the terminal.
 
-![Terminal logs from resetting the number](/images/builders/build/eth-api/libraries/ethersrs/ethersrs-4.webp)
+![Terminal logs from resetting the number](/images/builders/ethereum/libraries/ethersrs/ethersrs-4.webp)
 
-You can view the [complete script on GitHub](https://raw.githubusercontent.com/moonbeam-foundation/moonbeam-docs/master/.snippets/code/builders/build/eth-api/libraries/ethers-rust/deploy-contract/main.rs){target=\_blank}.
+You can view the [complete script on GitHub](https://raw.githubusercontent.com/moonbeam-foundation/moonbeam-docs/master/.snippets/code/builders/ethereum/libraries/ethers-rust/deploy-contract/main.rs){target=\_blank}.
```

Comment:
This link is broken for me but maybe it will be all set once this is merged into master.